$OpenBSD: patch-parser-specs_config_spec,v 1.2 2012/12/17 08:51:12 dcoppa Exp $

Workaround for an error caused by xcb-util-0.3.6
In file included from src/config_parser.c:49:
include/GENERATED_config_enums.h:70: error: 'FONT' redeclared as different kind of symbol
/usr/X11R6/include/xcb/xcb_atom.h:60: error: previous declaration of 'FONT' was here

Fix bind[code|sym] --release
(upstream git commit 6dfb08170cb1b0bd1875b80c85fbc225ff22274d)

--- parser-specs/config.spec.orig	Mon Dec 17 09:31:48 2012
+++ parser-specs/config.spec	Mon Dec 17 09:33:33 2012
@@ -20,7 +20,7 @@ state INITIAL:
   'set'                                    -> IGNORE_LINE
   bindtype = 'bindsym', 'bindcode', 'bind' -> BINDING
   'bar'                                    -> BARBRACE
-  'font'                                   -> FONT
+  'font'                                   -> WINDOW_FONT
   'mode'                                   -> MODENAME
   'floating_minimum_size'                  -> FLOATING_MINIMUM_SIZE_WIDTH
   'floating_maximum_size'                  -> FLOATING_MAXIMUM_SIZE_WIDTH
@@ -264,12 +264,14 @@ state EXEC:
       -> call cfg_exec($exectype, $no_startup_id, $command)
 
 # font font
-state FONT:
+state WINDOW_FONT:
   font = string
       -> call cfg_font($font)
 
 # bindsym/bindcode
 state BINDING:
+  release = '--release'
+      ->
   modifiers = 'Mod1', 'Mod2', 'Mod3', 'Mod4', 'Mod5', 'Shift', 'Control', 'Ctrl', 'Mode_switch'
       ->
   '+'
@@ -313,6 +315,8 @@ state MODE_IGNORE_LINE:
       -> MODE
 
 state MODE_BINDING:
+  release = '--release'
+      ->
   modifiers = 'Mod1', 'Mod2', 'Mod3', 'Mod4', 'Mod5', 'Shift', 'Control', 'Ctrl', 'Mode_switch'
       ->
   '+'
